src/nidcpower/system_tests/test_system_nidcpower.py

```diff
@@ -175,8 +175,8 @@ def test_measure(self, session):
     @pytest.mark.channels('0')
     def test_query_output_state(self, session):
         with session.initiate():
-            assert session.query_output_state(nidcpower.OutputStates.VOLTAGE) is True   # since default function is DCVolt when initiated output state for DC Volt\DC current should be True and False respectively
-            assert session.query_output_state(nidcpower.OutputStates.CURRENT) is False
+            assert session.query_output_state(nidcpower.OutputStates.CONSTANT_VOLTAGE) is True   # since default function is DCVolt when initiated output state for DC Volt\DC current should be True and False respectively
```

Comment:
I want to make a note that while we could update this test to also exercise the new enum value, it would require updating the driver version on the VM, first, and it's not necessary in the first place to test the function.

That's just a note for anyone who looks at this PR later.